Containerfile.aws-load-balancer-operator-bundle

```
@@ -32,12 +32,32 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=aws-load-balancer-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.3,stable-v1
+LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.2,stable-v1
```

```suggestion
LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.3,stable-v1
```
Since we added the new channel and bumped the bundle to 1.3.0 in #213.